lib/grape_entity/entity.rb

```diff
@@ -557,7 +557,7 @@ def delegate_attribute(attribute)
       if respond_to?(name, true)
         send(name)
       else
-        if object.respond_to?(name, true)
+        if object.respond_to?(name, true) && object.method(name).arity == 0
```

Comment:
This worries me. If there's a special case with `Hash` I would just check for that.